content/post/2019-08-05_ggplot2-tutorial.Rmd

```diff
@@ -18,92 +18,99 @@ knitr::opts_chunk$set(warning = FALSE, message = FALSE, out.width = "100%",
+Now it finds its proper place on this homepage!
+(Plus I added a ton of other updates---just to name a few: The fantastic `{patchwork}`, `{ggtext}` and `{ggforce}` packages. How to deal with custom fonts and colors. A collection of R packages tailored to create interactive charts. And several other chart types including pie charts because everyone looooves pie charts!)
 
-Since the blog entry by Zev Ross was not updated for some years and step by step this became a unique version of a tutorial, I decided to host the updated version on my GitHub. Now it finds its proper place on this homepage! (Plus I added a ton of other updates—just to name a few: The fantastic `{patchwork}`, `{ggtext}` and `{ggforce}` packages. How to deal with custom fonts and colors. A collection of R packages tailored to create interactive charts. And several other chart types including pie charts because everyone looooves pie charts!)
+<img src="/img/ggplot-tutorial/overview.png" style="height: 100%; width: 100%; object-fit: contain" onclick="window.open(&apos;https://www.cedricscherer.com/img/ggplot-tutorial/overview.png&apos;, &apos;blank&apos;);"/>
 
-<img src="/img/ggplot-tutorial/overview.png" style="height: 100%; width: 100%; object-fit: contain" onclick="window.open('https://www.cedricscherer.com/img/ggplot-tutorial/overview.png', 'blank');"/>
 <figcaption>Some exemplary plots included in this tutorial.</figcaption>
 
 Major changes I've made:
 
-* to follow the R style guide (e.g. by [Hadley Wickham](http://adv-r.had.co.nz/Style.html), [Google](https://google.github.io/styleguide/Rguide.xml) or the [Coding Club](https://ourcodingclub.github.io/2017/04/25/etiquette.html#syntax) style guides),
-* to change style and aesthetics of plots (e.g. axis titles, legends and nice colors for all plots not only some),
-* to have a updated version which keeps track of changes in `{ggplot2}` (current version: 3.4.0),
-* to modify data import (GitHub source),
-* to add additional tips on a vast range of topics, including for example chart choice, color palettes, modifying titles, adding lines, modifying legends, annotations with labels, arrows and boxes, multi-panel plots, interactive visualizations, ...
-
-
+-   to follow the R style guide (e.g. by [Hadley Wickham](https://style.tidyverse.org), [Google](https://google.github.io/styleguide/Rguide.xml) or the [Coding Club](https://ourcodingclub.github.io/2017/04/25/etiquette.html#syntax) style guides),
+-   to change style and aesthetics of plots (e.g. axis titles, legends and nice colors for all plots not only some),
+-   to have a updated version which keeps track of changes in `{ggplot2}` (current version: 3.4.0),
+-   to modify data import (GitHub source),
+-   to add additional tips on a vast range of topics, including for example chart choice, color palettes, modifying titles, adding lines, modifying legends, annotations with labels, arrows and boxes, multi-panel plots, interactive visualizations, ...
```

Comment:
Maybe add a bullet here to talk about `..level..` syntax since it is still very visible in the wild.